distributions/Dist_dl3185_2.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 from base_distribution import BaseDistribution
 
-class Dist_kc90(BaseDistribution):
+class Dist_dl3185_2(BaseDistribution):
```

Comment:
It looks like you renamed Dist_kc90 to Dist_dl3185_2. That will delete my distribution. Can you
0. copy your Dist_dl3185_2 file to a temporary location 
1. change branch to master
2. create a new branch from that (i.e.. start over)
3. change to that new branch
4. copy your Dist_dl3185_2 file into the distributions directory
5. commit, push/sync to GitHub, create new pull request